.github/workflows/github-actions.yml

```
@@ -23,16 +23,36 @@ jobs:
         run: ./gradlew build
 
       - name: Test with Gradle
+        id: test
         run: ./gradlew test
+        if: always()
 
       - name: Publish Test Report
         uses: mikepenz/action-junit-report@v3
         if: always() # always run even if the previous step fails
```

このコメントなくても大丈夫だと思います。
```suggestion
        if: always()
```

---

コメントを消しました。